cloud-build/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# This Dockerfile spins up a container where presubmit tests are run.
+# Cloud Build orchestrates this process.
+
+FROM gcr.io/cloud-dataproc-ci/init-actions-image
```

Comment:
How this image is created?

Reply:
I created it by manually running cloud build steps that pushed the image created from the Dockerfile.

Reply:
Can we depend on default gcloud image instead? Or this image has some pre-installed software (fastunit, etc)? In general I think that we shouldn't depend on one-off images and instead put all boot strapping logic in this image.

Reply:
This image has fastunit pre-installed. I fixed the Dockerfile to base itself off the gcloud image and also included the fastunit directory so that it can be readily used.

Reply:
The downside is that the gcloud image doesn't have fastunit pre-installed so we have to copy the fastunit directory.

Reply:
I was thinking about cloning fastunit repo in Docker file as an alternative - added separate comment for this.